README.md

```diff
@@ -1,17 +1,11 @@
 # concordances-rw-dynamodb
-_Should be the same as the github repo name but it isn't always._
-
 [![Circle CI](https://circleci.com/gh/Financial-Times/concordances-rw-dynamodb/tree/master.png?style=shield)](https://circleci.com/gh/Financial-Times/concordances-rw-dynamodb/tree/master)[![Go Report Card](https://goreportcard.com/badge/github.com/Financial-Times/concordances-rw-dynamodb)](https://goreportcard.com/report/github.com/Financial-Times/concordances-rw-dynamodb) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/concordances-rw-dynamodb/badge.svg)](https://coveralls.io/github/Financial-Times/concordances-rw-dynamodb)
 
 ## Introduction
 
-_What is this service and what is it for? What other services does it depend on_
-
-Reads / Writes concorded concepts to DynamoDB
+This service reads and writes concorded concepts to DynamoDB
```

Comment:
This is not really true as not concepts are written. Perhaps use the overview in the api document instead? https://docs.google.com/document/d/1vyXZOJrj19KS6uHD2jBx1DOO4PesAjh043034AXR72o/edit# I might additionally add what "a concordance" is i.e. A concordance is linking one primary concept identifier to another concept identifier